scripts/build-and-test.sh

```diff
@@ -54,8 +60,7 @@ if [[ $VARIANT == "coverage" ]]; then
   jobs
   # Kill it gracefully
   kill -INT %1
-  sleep 1
-  kill -INT %1 || echo "Dead already"
+  wait
```

Comment:
One thing to note, which may or may not be pertinent, is that the travis run for this pull request didn't actually run `wstest`, even though it started the echo server.